packages/mui-material-next/src/Switch/Switch.types.ts

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
-import { OverridableStringUnion } from '@mui/types';
-import { InternalStandardProps as StandardProps, Theme } from '@mui/material';
-// eslint-disable-next-line no-restricted-imports
+import { OverridableStringUnion, OverrideProps } from '@mui/types';
+import { Theme } from '@mui/material/styles';
+import { InternalStandardProps as StandardProps } from '@mui/material/index';
```

Comment:
```suggestion
import { InternalStandardProps as StandardProps } from '@mui/material';
```

No need for index.

Reply:
@mnajdova I have fixed this with an eslint-disable. I think we can merge it now.